webpack.flavour.config.js

```diff
@@ -0,0 +1,17 @@
+const entry = require('./package.json').main
+
+module.exports = {
+  entry,
+  target: 'node',
+  mode: 'none',
+  output: {
+    filename: 'index.js'
+  },
+  module: {
+    // to ignore the warnings like :
+    // WARNING in ../libs/node_modules/bindings/bindings.js 76:22-40
+    // Critical dependency: the request of a dependency is an expression
+    // Since we cannot change this dependency. I think it won't hide more important messages
```

Comment:
Weeeell why wouldn't it hide more important messages though? I think i'd rather keep those warnings.

Reply:
That's actually a shorter version of `webpack.config.js`, I kept the comment. But you're right we may remove this configuration and keep the warnings.

Reply:
(Done)